yarn.lock

```diff
@@ -6247,6 +6252,11 @@ jest@^27.4.3:
     import-local "^3.0.2"
     jest-cli "^27.5.1"
 
+jiff@^0.7.3:
```

Comment:
We don't appear to use this, so it's probably worth getting rid of by re-generating `yarn.lock`.